tests/fixtures/group.py

```diff
@@ -51,3 +51,10 @@ def active_group(timestamp, availability_slots):
     group.coordinators_former.clear()
     group.save()
     yield group
+
+
+@pytest.fixture
+def pending_group(active_group):
+    active_group.project_status = GroupProjectStatus.PENDING
```

Comment:
I also not sure about this, if do not use a active group my tests fail because `students/teachers/coordinators` filtered by active statuses so I assumed that if someone in the group he/she is in an active status though this might be not true for pending groups

Reply:
A pending group is an attempt at creating a "really working" group. So it makes sense that even a pending group is composed of people that are active in the project as a whole. It doesn't make any sense to create a group (and any group is pending in the beginning) with people that are not active within the whole project.